packages/cli/yaml/validator/src/rules/valid-example-endpoint-call/validateResponse.ts

```diff
@@ -26,15 +26,44 @@ export function validateResponse({
     file: FernFileContext;
     workspace: FernWorkspace;
     errorResolver: ErrorResolver;
+}): RuleViolation[] {
+    if (example == null) {
+        return [{ severity: "warning", message: "Response example is missing." }];
+    }
```

Comment:
@abvthecity why did you add this?

Reply:
ah this is probably not needed now that i'm reading the original code.
`return [];`